programs/drift/src/controller/orders.rs

```diff
@@ -1720,12 +1720,15 @@ fn fulfill_perp_order(
             .fuel_numerator(user, now),
         )?;
 
-    user_stats.update_fuel_bonus(
-        taker_margin_calculation.fuel_deposits,
-        taker_margin_calculation.fuel_borrows,
-        taker_margin_calculation.fuel_positions,
-    )?;
-    user.last_fuel_bonus_update_ts = now;
+    // user hasnt recieved initial fuel or below global start time
+    if user.last_fuel_bonus_update_ts != 0 || now > FUEL_START_TS {
```

Comment:
id just put this check in the actual update_fuel_bonus code so there is less duplicate code?